README.md

```diff
@@ -112,11 +112,59 @@ FUNCTION F_SetStateParams : BOOL
 END_FUNCTION
 ```
 
-If you want to use the tokenized source for your own purposes, you'll have
-to dig into the source code from there.
+It is also possible to parse the source code into a tokenized SourceCode tree:
+
+```python
+>>> import blark
+>>> blark.parse_source_code(
+...     """
+...                 PROGRAM ProgramName
+...                 VAR_INPUT
+...                     iValue : INT;
+...                 END_VAR
+...                 VAR_ACCESS
+...                     AccessName : SymbolicVariable : TypeName READ_WRITE;
+...                 END_VAR
+...                 iValue := iValue + 1;
+...             END_PROGRAM
+... """
+... )
+SourceCode(items=[Program(name=Token('IDENTIFIER', 'ProgramName'), declarations=[InputDeclarations(attrs=None, items=[VariableOneInitDeclaration(variables=[DeclaredVariable(variable=SimpleVariable(name=Token('IDENTIFIER', 'iValue'), dereferenced=False), location=None)], init=TypeInitialization(indirection=None, spec=SimpleSpecification(type=Token('DOTTED_IDENTIFIER', 'INT')), value=None))]), AccessDeclarations(items=[AccessDeclaration(name=Token('IDENTIFIER', 'AccessName'), variable=SimpleVariable(name=Token('IDENTIFIER', 'SymbolicVariable'), dereferenced=False), type=DataType(indirection=None, type_name=Token('DOTTED_IDENTIFIER', 'TypeName')), direction=Token('READ_WRITE', 'READ_WRITE'))])], body=StatementList(statements=[AssignmentStatement(variables=[SimpleVariable(name=Token('IDENTIFIER', 'iValue'), dereferenced=False)], expression=BinaryOperation(left=SimpleVariable(name=Token('IDENTIFIER', 'iValue'), dereferenced=False), op=Token('ADD_OPERATOR', '+'), right=Integer(value=Token('INTEGER', '1'), type_name=None)))]))], filename=PosixPath('unknown'), raw_source='\n                PROGRAM ProgramName\n                VAR_INPUT\n                    iValue : INT;\n                END_VAR\n                VAR_ACCESS\n                    AccessName : SymbolicVariable : TypeName READ_WRITE;\n                END_VAR\n                iValue := iValue + 1;\n            END_PROGRAM\n')
+```
+
+Alternatively, if you only want the tree:
+
+```python
+In [1]: import blark
+
+In [2]: parser = blark.parse.new_parser(start="function_block_body")
+
+In [3]: parser.parse(
+    ...:     """// Default return value to TRUE
+    ...: SerializeJson := TRUE;
+    ...:
+    ...: // Set to Root of Structure
+    ...: Root();
+    ...:
+    ...: SerializeJson := SerializeJson AND _serializedContent.Recycle();
+    ...:
+    ...: // Set up Initial States for Indices
+    ...: lastLevel := Current.LEVEL;
+    ...: outerType := Current.JSON_TYPE;
+    ...: """
+    ...: )
+Out[3]: Tree(...)
+```
+
+For some additional reference regarding this syntax, refer to
+[the comment here on issue #20](https://github.com/klauer/blark/issues/20#issuecomment-1099699641)
```

Comment:
I did write a nice little comment there 😁 

Maybe that comment should be reformatted as part of the docs or the wiki. I'll make an issue to put some sphinx docs in the repo. (I usually do it by default with my other projects so I'm a bit surprised to not even find a `docs/` directory here...)